app/models/manageiq/providers/openstack/network_manager/refresh_parser.rb

```diff
@@ -330,16 +336,6 @@ def parse_floating_ip(ip)
     # Helper methods
     #
```

Comment:
I'd get rid of this `# Helper methods` comment, in the end all private methods are helpers :grinning: